### DISCUSSION TITLE
Themes Rename

### DISCUSSION THREAD

**Discussion by hishamco** [OrchardCMS/OrchardCore#11301] [Ideas]
I'm not sure if I'm the only one  - but probably not - notice that our themes are not named properly, so can we:

1. Prefix all the theme with `OrchardCore`
2. Make theme names consistent, so we should rename `TheAdmin` & `SafeMode` to `TheAdminTheme` & `TheSafeModeTheme`

Another suggestion is to prefix them with `OrchardCore.Themes` something similar to the translations

**Comment by Skrypt** [comment]
I kind of agree that the theme names are silly. OrchardCore.Themes.TheAdmin would totally make sense. This would make them identifiable instead of getting assemblies with TheAdminTheme.dll for example which doesn't give any hint about where it comes from.

**Comment by hishamco** [comment]
I know it's a breaking changes, but we can do it for `2.0.0`. It's also useful for NuGet package discovery, me as I user I can't guess what the themes names for OC are. @sebastienros if you agree on the this I can start a PR for this and merge it once the `2.0.0` is ready
